feedstocks.html.tmpl

```diff
@@ -39,27 +39,27 @@
                 <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-main-collapse">
                     <i class="fa fa-bars"></i>
                 </button>
-                <a class="navbar-brand page-scroll" href="./#page-top">
+                <a class="navbar-brand page-scroll" href="https://conda-forge.github.io/#page-top">
                     <i><img src="img/anvil_white.png" alt="An anvil" style="height: 1em;"></i>  <span class="light">conda-forge</span>
```

Comment:
Missed fixing this link to be absolute. Correcting it in PR ( https://github.com/conda-forge/conda-forge.github.io/pull/468 ).